### ISSUE TITLE
Lab 11 Task 04

### ISSUE THREAD

**Issue by shawnwiggins** [ccsf-math-108/materials-sp24#12]
A student reported:

> I've gone through lab 11 task 4, and I'm finding a mismatch between the results in my table and what is listed under the section "Example Point":
> 
> Let's use the first penguin listed in the test table as an example to use for a few tasks. That penguin should have a culmen length of 52.0 mm and a flipper length of 197 mm. You will now make and use a kNN classifier to predict the Adélie label for that penguin.
> 
> The penguin with those attributes is in the middle of my training set (so not right on the border). My test set starts with an Adelie/True, Culmen Length/43.1, Flipper Length/197.
> 
> I can't figure out where I'm going wrong - I do pass the grader, but it's only checking the size of the two sets, not the content.
> 
> This can wait until class to resolve, I'm just wondering if there is something wrong with np.seed(2024) not working as expected?

The issue is that the description under "An Example Point" does not match the first row from the random test set generated.

- [x] Update the culmen length and label so that the description reads as: 

    > Your goal is to be able to classify a penguin as Adélie or not based on the labeled data in Gorman's data set. Let's use the first penguin listed in the `test` table as an example to use for a few tasks. That penguin should have a culmen length of **43.1 mm** and a flipper length of 197 mm. You will now make and use a kNN classifier to predict the Adélie label for that penguin. You will be focusing on using table rows to do this. We know the true label for this penguin is **`True`**, but for now, let's ignore it and see how the classifier works. So an unlabeled example row for that penguin would be:

**Comment by shawnwiggins** [comment]
The description should be updated. If you click on the link in Canvas to Lab 10, then the changes should be imported into your notebook. I'll leave this issue open until the due date for this assignment.
